kafka-ui-react-app/src/components/common/Input/InputIcon.styled.ts

```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components';
+import { Colors } from 'theme/theme';
+
+interface Props {
+  className: string;
+  position: 'left' | 'right';
+  inputSize: 'M' | 'L';
+}
+
+const StyledIcon = styled.i<Props>`
+  position: absolute;
+  top: ${(props) => (props.inputSize === 'M' ? '8px' : '12px')};
+  left: ${(props) => (props.position === 'left' ? '12px' : '97%')};
```

Comment:
You can `unset` left and properly set `right` positioning the same way

Reply:
Don't quite get what you mean here 🤔

Reply:
As I understand this piece of code you decide the side of the icon with this prop 'position'. If it's left, the icon is on the left side, else on the right one.
I decide to use more precise positioning
```
left: ${(props) => (props.position === 'left' ? '12px' : 'unset')}; 
right: ${(props) => (props.position !== 'left' ? '12px' : 'unset')};
````